### PR TITLE
GString support

### PR DESCRIPTION
**Pull request by philn**

### CODE REVIEW

File: src/auto/pixbuf.rs [hunk 1]
```diff
@@ -121,7 +120,7 @@ impl Pixbuf {
     }
 }
 
-pub trait PixbufExt {
+pub trait PixbufExt: 'static {
```
Comment (GuillaumeGomez) [inline src/auto/pixbuf.rs]: Is the `'static` normal?

Comment (sdroege) [reply src/auto/pixbuf.rs]: Yes

**Comment by sdroege** [comment]
@GuillaumeGomez Please merge :)

**Comment by GuillaumeGomez** [comment]
Yes sir!

Thanks @philn!

**Comment by philn** [comment]
Hum yes, I believe there's a few more ;) I'll double-check tomorrow.

**Pull request merged**